### ISSUE TITLE
[Epic-1] Apply Time Independent TT Weight via Normal Forward-Integration Backward Gradient Descent Framework

### ISSUE THREAD

**Issue by mbaddar1** [mbaddar1/phd#85]

**Comment by mbaddar1** [comment]
useful
https://refubium.fu-berlin.de/bitstream/handle/fub188/3366/dissertation_pg_final.pdf?sequence=1

**Comment by mbaddar1** [comment]
ALS + Backprop idea (Hyprid linear, non-linear trajectory) 
http://cs231n.stanford.edu/slides/2017/cs231n_2017_lecture4.pdf    slide 26
